src/Reflex/Dom/Contrib/Widgets/ButtonGroup.hs

```diff
@@ -167,15 +167,15 @@ radioGroup dynName dynEntryList cfg = do
      <> bool mempty ("checked" =: "checked") chkd
     handleOne _ dynV dynChecked = do
 
-      el "tr" $ do
+      el "tr" $ el "td" $ el "label" $  do
```

Comment:
The splitting of the little radio  circle and the text into two different table columns is rather uncommon. I think,  Haskell and Reflex should do these buttons in a more standard way.